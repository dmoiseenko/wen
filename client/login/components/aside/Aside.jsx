import React from 'react';


export default function Aside() {
  return (
    <div className="column is-8 is-hidden-mobile hero-banner">
      <section className="hero is-fullheight is-dark">
        <div className="hero-body">
          <div className="container section">
            <div className="has-text-right">
              <h1 className="title is-1">The secret of getting ahead is getting started.</h1> <br />
              <p className="title is-3">Mark Twain</p>
            </div>
          </div>
        </div>
        <div className="hero-foot">
          <p className="has-text-centered">Image Â© Glenn Carstens-Peters via unsplash</p>
        </div>
      </section>
    </div>
  );
}
