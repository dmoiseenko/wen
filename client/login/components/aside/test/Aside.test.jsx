import React from 'react';
import renderer from 'react-test-renderer';

import Aside from '../Aside';


describe('<Aside />', () => {
  const props = {};

  it('should render', () => {
    const tree = renderer.create(
      <Aside {...props} />
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
