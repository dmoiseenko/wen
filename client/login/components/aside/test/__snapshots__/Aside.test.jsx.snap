// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Aside /> should render 1`] = `
<div
  className="column is-8 is-hidden-mobile hero-banner"
>
  <section
    className="hero is-fullheight is-dark"
  >
    <div
      className="hero-body"
    />
    <div
      className="hero-foot"
    >
      <div
        className="has-text-right"
      >
        <h1
          className="title is-2"
        >
          The secret of getting ahead is getting started.
        </h1>
        <br />
        <p
          className="title is-4"
        >
          Mark Twain
        </p>
      </div>
      <p
        className="has-text-centered"
      >
        Image Â© Erik Heddema via unsplash
      </p>
    </div>
  </section>
</div>
`;
